script.js

```diff
@@ -2,344 +2,485 @@ const playButton = document.getElementById('play');
+    gender: 'Female',
+    id: 33
   },
   {
     apiName: "Spanish Female",
-    flag: '🇪🇸',
-    gender: 'Female'
+    accent: 'Spanish',
+    flag: 'https://flagcdn.com/es.svg',
+    gender: 'Female',
+    id: 34
   },
   {
     apiName: "Spanish Latin American Female",
-    flag: '🇲🇽',
-    gender: 'Female'
+    accent: 'Spanish Latin American',
+    flag: 'https://flagcdn.com/mx.svg',
+    gender: 'Female',
+    id: 35
   },
   {
     apiName: "Spanish Latin American Male",
-    flag: '🇲🇽',
-    gender: 'Male'
+    accent: 'Spanish Latin American',
+    flag: 'https://flagcdn.com/mx.svg',
+    gender: 'Male',
+    id: 35
   },
   {
     apiName: "Swedish Female",
-    flag: '🇸🇪',
-    gender: 'Female'
+    accent: 'Swedish',
+    flag: 'https://flagcdn.com/se.svg',
+    gender: 'Female',
+    id: 36
   },
   {
     apiName: "Swedish Male",
-    flag: '🇸🇪',
-    gender: 'Male'
+    accent: 'Swedish',
+    flag: 'https://flagcdn.com/se.svg',
+    gender: 'Male',
+    id: 36
   },
   {
     apiName: "Turkish Female",
-    flag: '🇹🇷',
-    gender: 'Female'
+    accent: 'Turkish',
+    flag: 'https://flagcdn.com/tr.svg',
+    gender: 'Female',
+    id: 37
   },
   {
     apiName: "Turkish Male",
-    flag: '🇹🇷',
-    gender: 'Male'
+    accent: 'Turkish',
+    flag: 'https://flagcdn.com/tr.svg',
+    gender: 'Male',
+    id: 37
   },
   {
     apiName: "Ukrainian Female",
-    flag: '🇺🇦',
-    gender: 'Female'
+    accent: 'Ukrainian',
+    flag: 'https://flagcdn.com/ua.svg',
+    gender: 'Female',
+    id: 38
   },
   {
     apiName: "Vietnamese Female",
-    flag: '🇻🇳',
-    gender: 'Female'
+    accent: 'Vietnamese',
+    flag: 'https://flagcdn.com/vn.svg',
+    gender: 'Female',
+    id: 39
   },
   {
     apiName: "Vietnamese Male",
-    flag: '🇻🇳',
-    gender: 'Male'
+    accent: 'Vietnamese',
+    flag: 'https://flagcdn.com/vn.svg',
+    gender: 'Male',
+    id: 39
   }
 ];
 
-// options
+// Flags
 const selectFlag = document.getElementById('flags');
-const genderList = document.getElementById('gender');
-
-const onlyFlagsUnique = new Set(availableVoices.map(pl => pl.flag));
+const onlyFlagsUnique = new Set(availableVoices.map(f => f.flag));
+let I = 0;
 
 onlyFlagsUnique.forEach(flag => {
-  const optionElement = document.createElement("option");
-  optionElement.text = flag;
-  optionElement.value = flag;
+  I++;
+  const optionElement = document.createElement("li");
+  optionElement.setAttribute("class", "flagList");
+  optionElement.setAttribute("onclick", "handleClick("+I+")");
+  const imgElment = document.createElement("img");
+  imgElment.setAttribute("class", "imgList");
+  imgElment.src = flag;
+  // imgElment.alt = flag.accent;
+  optionElement.appendChild(imgElment);  
   selectFlag.appendChild(optionElement);
 });
 
+const handleClick = (fi) => { // fi = Flag ID
+  console.log(fi);
+};
+
+// Scroll Flags
+const goUp = document.getElementById('up');
+const goDown = document.getElementById('down');
```

Comment:
These names aren't good enough, when you say 'go' is usually to execute an action, not to declare a var name.